tools/CMakeLists.txt

```diff
@@ -53,6 +45,7 @@ if (WIN32)
 endif()
 
 target_link_libraries(gz
+ CURL::libcurl
```

Comment:
I don't see any usage of `curl` by `gz`; I would remove this line

I'm guessing the cmake code you've removed in this file is vestigial

Reply:
dba62f3